### ISSUE TITLE
Contrast too low

### ISSUE THREAD

**Issue by jhayley** [soft-aesthetic/soft-era-vs-code#2]
Hi. I just started using this theme and I love it. I've found a couple spots that have extremely contrast and are difficult to read.

The contrast on the comments is slightly too low for me, right on the border of needing to squint to see them. Compare this theme (picture 1) with the current "soft" theme I use, quiet light (2). Another less important nicety is having a special, even higher contrast color for various docstring formats (as you can see quiet light implements this for the triple forward slash, not sure if that's the only one that vscode allows cutsom formatting for though).

![image](https://user-images.githubusercontent.com/1892337/38743543-2da7a734-3ef4-11e8-87e9-2175a84c83aa.png)
![image](https://user-images.githubusercontent.com/1892337/38743610-635b6df2-3ef4-11e8-8cfa-f8bf05d7520d.png)

Another place with extremely low contrast is in the autocomplete menus, especially the doc popup on the right:

![image](https://user-images.githubusercontent.com/1892337/38748472-0c62b33e-3f03-11e8-825b-0c77e24d2f34.png)

Thanks ^~^ (updated to show new places with low contrast)

**Comment by animalphase** [comment]
@jhayley awesome thank you for this analysis!! i'm testing a few default text color values to increase contrast. i felt like in Atom the color was as light as it could be, and the rendering in other editors/IDEs is not as weighted and even harder to read. these screenshots help a ton!!!

**Comment by animalphase** [comment]
@jhayley testing a value `#ba989c` for default text with the latest patch… still testing a final value across platforms, but VS Code text rendering is really light and def could use a contrast boost ASAP :)

Leaving this issue open until there's a final hex value decision across platforms.

**Comment by jhayley** [comment]
Great thanks! My initial impressions that this is pretty significantly better for the dropdowns. With this change I'll be using it now so I'll let you know if I have any issues!

**Comment by 76784** [comment]
My compliments on your very nice theme, "soft era". I am a fan of low contrast themes, and I created one called Daobeam. I'd be curious to see what everyone thinks of it. Please check it out if you're interested. Thanks, and happy coding.

**Comment by jhayley** [comment]
@76784 please dont come into other repo's issues to advertise your own projects

**Comment by 76784** [comment]
Didn't mean to offend. Was more trying to look for and participate in a community. But I can see where that would be thought of as bad form. My apologies.

**Comment by animalphase** [comment]
@76784 thanks for being kind & listening :) yah i think especially in issue threads they're supposed to be on-topic for ppl looking to understand the issue or get a to-do task. plus ppl participating/watching the thread get notifications from it. i'm down to talk theme design and share stuff if you want to hit me up on email or whatever~ (can find me thru my profile)

and @jhayley thanks for keeping us on topic ⭐️

**Comment by animalphase** [comment]
@maxmechanic that would be helpful! with this theme, i was specifically trying to create a low-contrast theme (because that is what my eyes need). so i'm not sure how much i *personally* want to follow that… however, if it looks like we can develop a good color balance and aesthetic, that passes AA, and is still usable for me……… that would be a good goal :))

**Comment by lovemecomputer** [comment]
@jhayley closing this issue as of version 1.0!!! thank u for documenting this all those months back

![image](https://user-images.githubusercontent.com/3846271/52234822-30bf0880-2888-11e9-80fd-4ebcfe0b0b9b.png)

**Comment by phganh** [comment]
the theme is lovelyyyyy. i found out about your theme today and start using it. it's just about the contrast that sometime quite hard to follow the code or look at it from a bit further (i use laptop and sometime i put it on the stand). anyway, thank you for creating such a nice theme.
